src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Details.cshtml

```diff
@@ -5,6 +5,7 @@
 
 @{
     ViewData["Title"] = Model.PageTitle;
+    var isReadOnly = Model.WithParty != SFA.DAS.CommitmentsV2.Types.Party.Employer;
```

Comment:
Can we add this as a property the view model, so that we can avoid code in the razor and cover it with unit tests?